bin/amt-cli

```diff
@@ -20,6 +20,7 @@ prog
   .command("generate <output>")
   .option("-c, --count", "The number of rows to generate", 10e3 + 1)
   .option("-e, --email", "The email provider used on generated email addresses", "gomerits.testing.com")
+  .option("-s, --headers", "The comma separated list of headers to generate", "email,firstName,lastName,city,state,zipCode,licenseNumber,dateIssued,expirationDate,suspended,revoked,revokeReason,longText")
```

Comment:
is the goal of this option to just allow someone to specific less headers than what we have by default?